test/parallel/test-path.js

```diff
@@ -406,6 +422,11 @@ assert.strictEqual(path.win32.normalize('a//b//.'), 'a\\b');
 assert.strictEqual(path.win32.normalize('//server/share/dir/file.ext'),
                    '\\\\server\\share\\dir\\file.ext');
 assert.strictEqual(path.win32.normalize('/a/b/c/../../../x/y/z'), '\\x\\y\\z');
```

Comment:
#### Optional:
Could you parameterize this part like the others are:
```js
[[a,b]].forEach(([tested, expected]) => ...
```
(not the scope of this PR but still would be nice)